lib/wab/impl/data.rb

```diff
@@ -37,27 +38,14 @@ def initialize(value, repair, check=true)
       # same as ['child', 'grandchild'].
       def get(path)
         if path.is_a?(Symbol)
-          node = @root[path] 
+          node = root[path]
         else
```

Comment:
You'll then able to `read` the value of `Data.root` outside `WAB::Impl::Data`. If there won't be a need for that, then, I'll revert this..